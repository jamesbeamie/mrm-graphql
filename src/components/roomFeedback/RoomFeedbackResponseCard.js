import React from 'react';
import PropTypes from 'prop-types';

/**
 * Represents the cards of data
 *
 * @param {Object} roomFeedbackCard
 *
 * @returns {JSX}
 *
 * @constructor
 */
const RoomFeedbackCard = ({ title, value }) => (
  <section>
    <p>{title}</p>
    <p>{value}</p>
  </section>
);

RoomFeedbackCard.defaultProps = {
  title: 'Title',
  value: 0,
};

RoomFeedbackCard.propTypes = {
  title: PropTypes.string,
  value: PropTypes.number,
};

export default RoomFeedbackCard;
