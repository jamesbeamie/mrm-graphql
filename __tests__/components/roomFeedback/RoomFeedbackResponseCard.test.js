import React from 'react';
import { shallow } from 'enzyme';
import RoomFeedbackResponseCard from '../../../src/components/roomFeedback/RoomFeedbackResponseCard';

const initProps = {
  title: '',
  value: 0,
};

describe('Room feedback Card Component', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<RoomFeedbackResponseCard {...initProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});
