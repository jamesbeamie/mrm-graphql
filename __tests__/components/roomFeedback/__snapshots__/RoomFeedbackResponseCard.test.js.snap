// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Room feedback Card Component should render correctly 1`] = `
<section>
  <p />
  <p>
    0
  </p>
</section>
`;
